### ISSUE TITLE
Ds_SecurePayment should be implemented (as optional)

### ISSUE THREAD

**Issue by JustDevZero** [ajostergaard/django-payments-redsys#11]
I think it would be interesting to implement the option of Ds_SecurePayment, since some banks and cards are beginning to force the implementation of this.

**Comment by ajostergaard** [comment]
PRs gratefully received (I am no longer using Redsys).

**Comment by txels** [comment]
Hi @JustDevZero 

This is a really old issue, but nonetheless I've been checking the Redsys documentation and I'm not sure what you expect the provider to do... `Ds_SecurePayment` is a value in the response data, our payment request makes no difference, and the way I see it it is informational as to how the payment was performed on the Redsys end, that we would get only once the payment has taken place.

https://pagosonline.redsys.es/desarrolladores-inicio/integrate-con-nosotros/parametros-de-entrada-y-salida/

What would you expect for the provider to do?

I am leaning towards closing it as "won't do".
